### PR TITLE
Add example file to convert tv to rdf

### PR DESCRIPTION
**Pull request by techytushar**
There was a `rdf_to_tv.py` file, i thought it would be nice to have `tv_to_rdf.py` also.
Signed-off-by: Tushar Mittal <chiragmittal.mittal@gmail.com>

### CODE REVIEW

File: examples/tv_to_rdf.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# Converts an tag/value file to RDF format.
```
Comment (pombredanne) [inline examples/tv_to_rdf.py]: Use """ docstring instead of # comments

File: examples/tv_to_rdf.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Usage: tv_to_rdf <tagvaluefile> <rdffile>
+
+if __name__ == '__main__':
```
Comment (pombredanne) [inline examples/tv_to_rdf.py]: Could you make the code below a function and move the import at the top?
And call that function in this `if __name__ == '__main__'` block instead?

**Comment by techytushar** [comment]
done the required changes, please review @pombredanne

**Comment by pombredanne** [comment]
Sorry for this late merge!

**Pull request merged**